ui/de-lib/src/main/java/org/iplantc/de/theme/base/client/admin/toolAdmin/ToolAdminDisplayStrings.properties

```diff
@@ -0,0 +1,71 @@
+#Dialog Window
+dialogWindowName = Tool Admin
+dialogWindowUpdateBtnText = Save/Update
+addToolSuccessText = Tool successfully added!
+updateToolSuccessText = Tool successfully updated!
+dialogWindowDeleteBtnText = Delete
+deleteToolSuccessText = Tool successfully deleted!
+confirmOverwriteTitle = Public App Overwrite?
+confirmOverwriteDangerZone = Do not update container settings that are in use by tools in public apps unless it is certain the new container settings will not break reproducibility for those apps.\
+  <br>If required, the `overwrite-public` flag may be used to update these settings for public tools.\
+  <br><br> The following apps are currently using this tool:
+confirmOverwriteBody = Do you wish to use the overwrite-public flag now?
```

Comment:
Ah ok, I wasn't sure if making it as close to the Swagger text was best or not.  I'll make it more general. Thanks!